sources/packages/web/src/components/layouts/aest/AESTApplicationSideBar.vue

```diff
@@ -73,13 +83,15 @@ export default {
             label: `Parent ${index + 1}`,
             // TODO: in figma this icon is PRO version
             icon: "user",
+            command: () => goToSupportingUser(supportingUser.supportingUserId),
```

Comment:
I missed to ask this in the previous PR,I assume it is font awesome icon should we use icon like icon: ['fas', 'user'] as we follow in other places to be explicit.

Reply:
Or this this because we are waiting for PRO version? is yes, still we can be explicit.

Reply:
actually, if we don't specify the type, by default it will take `fas`. but i will upate

Reply:
Yeah but we were being explicit everywhere, which was nice IMO.

Reply:
So, if there is a way to avoid the array I would prefer.
Does the below code works? Why did we went for this array in the first place?

![image](https://user-images.githubusercontent.com/61259237/155800699-3e975518-8b2a-4dda-abec-9fcd0642544c.png)

Reply:
as discussed, its not working